init.c

```diff
@@ -772,9 +772,6 @@ static int fixup_options(struct thread_data *td)
 			ret = warnings_fatal;
 		}
 
-		if (!fio_option_is_set(o, refill_buffers))
```

Comment:
What am I missing here, I don't see where else it's being set for this?

Reply:
I think it gets set over in `init_flags()` https://github.com/sitsofe/fio/blob/9164932ecbb3aa4b1538080f97452a2fdb3f02f1/init.c#L1112 and `init_flags()` is called before `fixup_options()` over in `add_job()` (see https://github.com/sitsofe/fio/blob/9164932ecbb3aa4b1538080f97452a2fdb3f02f1/init.c#L1359 ).

Reply:
init_flags() may set the REFILL_BUFFERS flag, but not o->refill_buffers.

Reply:
Ah you're right (e.g. in any case where you're not doing `compress_percentage`). OK let me drop that commit.